types/distributions/distributions-tests.ts

```diff
@@ -0,0 +1,17 @@
+import distributions from 'distributions';
```

Comment:
there's no default export so you need to use `import * as distributions from 'distributions'`

Looks like a bug in the linter skipped the error that should have arisen from this.